### DISCUSSION TITLE
Redirect apps to a canonical domain

### DISCUSSION THREAD

**Discussion by iBobik** [btcpayserver/btcpayserver#4384] [Ideas / Feature Requests]
When some app has frontend deployed to a different domain (for branding purposes) it is accessible from both urls - e.g. https://pos.example.com and https://btcpay.example.com/apps/123/pos

It would not be a big deal if link from admin UI would not target to a main domain - staff unintentionally sends customers link with a wrong domain. Then customers sees unbranded link in their browser and even in the wallet when using LNURL.

So I propose to create redirect:
https://btcpay.example.com/apps/123/pos → https://pos.example.com

**Comment by dennisreimann** [comment]
Thanks for the request. I've implemented this in #4391, but we need to discuss it as some things need to change for this to work as intended. See the comments in the pull request.

**Reply by dennisreimann** [reply]
@iBobik This will become available  with v1.8 soon.
